docs/website/packages/codegen/copy.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 2
+title: Copying
+description: codegen's copy function
+---
+
+The `copy` function is designed to facilitate the duplication of source code from one FHIR version to another.
+
+## Basic Example
+
+### Set Up Your Source and Target Directories
+
+Ensure that you have two directories: one for the source FHIR version and another for the target FHIR version. For example, you might have a directory structure like this:
+
+- `src/r4b` (source)
+- `src/r5` (target)
```

Comment:
The repo is actually setup the otherway around - I suggest that's how we document it.

See https://bonfhir.dev/docs/contributing#where-to-write-new-code for more details